### DISCUSSION TITLE
Page Expired on login from mobile device

### DISCUSSION THREAD

**Discussion by pxd303** [JulianPrieber/littlelink-custom#298] [Issues]
whenever I login to my littlelink-custom from my MOBILE DEVICE i get a "419 Page Expired" error but am indeed logged in. 

What is happening here?

**Comment by JulianPrieber** [comment]
I'm not entirely sure

**Reply by JulianPrieber** [reply]
Does this happen on every browser you use on your phone?

**Reply by pxd303** [reply]
Just my phone

**Reply by JulianPrieber** [reply]
Alright but what if you use a different browser on your phone?

**Reply by pxd303** [reply]
Im going to move to a fresh install on the new release and get back to you.

**Reply by pxd303** [reply]
on the new install everthing is fine until I go to edit a user getting this error now. 
Call to undefined function findFiles() (View: /home/u397169879/domains/zztap.me/public_html/resources/views/panel/edit-user.blade.php)
<img width="1313" alt="Screen Shot 2023-01-25 at 5 13 57 PM" src="https://user-images.githubusercontent.com/121210037/214723318-b4b678af-123c-42f7-9dcd-126bf159665b.png">

**Reply by JulianPrieber** [reply]
This error will be addressed in the next version.
I think its even fixed on the latest beta.

**Reply by pxd303** [reply]
I was wrong the issue from mobile still exists and I see other [laravel](https://stackoverflow.com/questions/52583886/post-request-in-laravel-error-419-sorry-your-session-419-your-page-has-exp) developers experiencing the same. Yes Ive tried several mobile devices

![Screenshot_20230126-140751](https://user-images.githubusercontent.com/121210037/214951088-8242a0fc-3dae-4fa7-a97d-d4abfad5f67d.png)

**Reply by pxd303** [reply]
Joined beta to see if youve already fixed it. Will let you know in like 5

**Reply by pxd303** [reply]
Still not working on beta. Ive move to an entirely different server tried different phones and mobile browsers

**Reply by JulianPrieber** [reply]
<img width="1313" alt="214723318-b4b678af-123c-42f7-9dcd-126bf159665b" src="https://user-images.githubusercontent.com/60265788/214954325-9c888102-e07f-4ea2-b66c-38efdef9f254.png">
This error?

**Reply by pxd303** [reply]
I fixed it. with 'secure' => env('SESSION_SECURE_COOKIE', false) in config/session.php the delete cache

**Reply by JulianPrieber** [reply]
huh

**Reply by pxd303** [reply]
No the OP error 419 Page Expired error  good to go now with config/session.php
